src/components/footer/example-footer-with-additional-logo.njk

```diff
@@ -0,0 +1,89 @@
366.152-.427.06-.03.121.335.06.548.213-.03.426-.335.396-.852.122-.091.213-.152.304-.152.122 0 .396.365.792.365.7 0 1.095-.578 1.156-.852.03-.213.03-.426-.091-.579.243.214.335.518.335.761 0 1.005-1.796 1.492-2.526 2.192-.03.03-.061.06-.092.06l-.852-.334-.06.182s0 .03-.062.152c-.09.274-.273.883-.395 1.218l-.03.122.06.03c-.487.67-.882 1.4-.974 2.16-.213-.212-.395-.364-.608-.547.03-.06.03-.152.03-.243a.936.936 0 0 0-.183-.579.49.49 0 0 0-.395-.182c-.152 0-.305.03-.396.152-.03.03-.06.06-.06.091-.031-.03-.062-.06-.092-.06-.274-.214-.609-.457-.822-.7a.456.456 0 0 0-.365-.153h-.365c-.122 0-.183.091-.335.213-.06.061-.274.213-.487.366.091.09.213.213.304.334.061.061.092.122.153.183a.664.664 0 0 0 .152-.122.588.588 0 0 1 .304-.091c.122 0 .274.091.365.243.061.122.761 1.005 1.309 1.613-.06.03-.122.03-.183.061-.243.091-.79.091-1.034.061s-.274-.091-.457.183c-.06.091-.152.213-.213.304-.091.091-.152.213.03.487.244.426.853 1.522 1.066 2.009.213.487.456.578.821.73.396.152 1.066.548 1.127.64l.03-.366c-.122-.122-.487-.243-.64-.304-.151-.061-.182-.122 0-.122.153 0 .64.183.975.304-.092-.365-.457-.791-.67-1.004-.274-.091-.578-.091-.67 0 0 .091-.03.183-.152.183-.121 0-.182-.092-.152-.183.03-.122.213-.152.152-.365-.03-.213-.304-.244-.456-.274-.244-.335-.517-.852-.64-1.065-.121-.213-.06-.335.031-.396.213-.152.64.091 1.492.152.304.03.67.03 1.065.061.121.974.517 1.917 1.887 2.283.67.152 1.37.67 1.613 1.308-2.74-.06-3.196 1.035-3.196 2.04 0 1.156.974 1.095.974 1.947 0 .761-2.313 1.066-3.013 1.157-.578.091-.944.213-1.217.517-.274.274-.579.548-1.096 1.065 0 0 .213.061.426.03l.396-.517c.121-.182.182-.304.274-.304.09 0 .03.152-.061.335l-.335.548c1.37.091 1.43-.426 1.43-.609 0-.213-.334-.335-.152-.487.183-.122.365-.03.518.152.152.183.213.335.578.183 0 0 2.952-1.218 3.317-1.4.335-.152.244-.304.03-.457-.212-.182-.243-.487-.243-.791 0-.883.548-1.187 1.491-1.157-.03.153-.03.305-.03.427 0 1.4 2.1 2.77 2.678 2.77.244 0 .609-.183.609.638.03.609.152 1.066.03 1.766-.121.7-.304.821-.76 1.126-.427.304-.822 1.247-1.005 1.43.183.122.396.152.548-.304.122-.396.274-.64.457-.73-.153.425-.305 1.034-.366 1.217.548.09 1.4-.396 1.4-.792 0-.365-.304-.334-.304-.608 0-.122.122-.244.304-.244.213 0 .396.183.396.244.243-.122.396-.366.396-.913 0-.213 0-.457-.03-.67 0 .03.03.06.06.091.091.152.274.244.487.274.183.03.396-.03.578-.152.213-.152.365-.335.396-.578 0-.03.03-.091.03-.122a.613.613 0 0 0-.09-.335.927.927 0 0 0-.397-.274c-.213-.06-.456 0-.67.152a.958.958 0 0 0-.395.518c-.03-.761-.091-1.491-.091-1.765.03-.426.06-.609-.091-.67-.153-.06-.305.183-.518.183-.213 0-.822-.578-.822-1.187 0-.609.64-1.37.61-2.222 0-.243.151-.396.243-.152.121.274.304.67.578.974-.183 1.035.974 2.039 1.917 1.308-.456.061-.791-.334-1.004-.76.091 0 .152.03.243.03h.153c.334.396 1.37.365 1.643-.64-.183.214-.548.366-.822.336.396-.305.609-.822.609-1.34 0-2.647-3.196-2.952-3.196-5.326 0-.882.792-1.46 1.31-1.46.212 0 .364.06.486.152-.213.213-.304.517-.304.913.091 1.887 3.165 1.796 3.226 2.709.548-1.279-1.765-.853-1.857-2.679Z"></path></svg>',
```

Comment:
Think we need a better logo, this one looks too big next to the ONS one to me. You could try just reducing the size of it

Reply:
done